### PR TITLE
Feat/onchain manager

### PR DESCRIPTION
**Pull request by huynguyenh**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

**Review by anhnh12 — DISMISSED** [review]

File: internal/txsubmitter/telemetry.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			}
+			// 2. status = 1 -> transfer success
+			if status == 1 {
+				// TODO: add log in this function only
+				s.RemoveTx(hash, chainId)
+				s.handleSuccessSubmittedTx(txn)
+				continue
+				// TODO: add to notification queue, can be done later
+			}
+			// 3. status = 2 -> transfer pending, continue
```
Comment (huynguyenh) [inline internal/txsubmitter/telemetry.go:70]: update comment
